test_ci_backend.sh

```diff
@@ -2,6 +2,11 @@
 
 set -e
 
+# run django migrations check to ensure that there are no migrations left to create
+cd src || exit 1
```

Comment:
move to after line 20 (where you already did a cd)

Reply:
Changed as requested.